<launch>
    <node respawn="true" pkg="joy" type="joy_node" name="joy_driver">
        <param name ="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.05"/>
    </node>

    <node pkg="joy_teleop" type="joy_handle.py" name="joy_handle">
        <param name ="linear_axis" value="4" type="int"/>
        <param name="angular_axis" value="3" type="int"/>
        <param name="linear_maxVel" value="0.3" type="double"/>
        <param name="angular_maxVel" value="0.8" type="double"/>
        <param name ="linear2angular_button" value="0" type="int"/>
        <param name="arm_axis_x_pitch" value="1" type="int"/>
        <param name="arm_axis_y_roll" value="0" type="int"/>
        <param name="arm_axis_z" value="7" type="int"/>
        <param name ="arm_axis_yaw" value="6" type="int"/>
        <param name="arm_xy_maxVel" value="1" type="double"/>
        <param name="arm_angular_maxVel" value="1" type="double"/>
        <param name="arm_z_vel" value="1" type="double"/>
    </node>
</launch>

        
        
        
        
        
        